habs_portal/templates/map.html

```diff
@@ -79,8 +79,11 @@
 
     this.views.tocView = new TOCView({
       el: $('#toc'),
-      categoryCollection: this.collections.categories,
-      stationCollection: this.collections.stations
+      parentCollection: this.collections.categories,
+      childCollection: this.collections.stations,
+      parentChildAssociation: function(parent,child) {
+        return _.indexOf(child.get('categories'),parent.get('id')) >= 0
```

Comment:
@lukecampbell This line (and the corresopnding one in stations.html) is the primary reason I went the parent-child route instead of the category-stations route.  I can define the parent-child relationship here in the view's init.  When I spin up the plotting TOC, this function will essentially be the inverse.